src/estimators/exact_set.py

```diff
@@ -53,6 +53,11 @@ def __contains__(self, x):
     """Return true if x is contained in the sketch."""
     return x in self._ids
 
+  def add_ids(self, x):
+    """Adds a list of ids to the sketch."""
+    for item_id in x:
+      self.add(item_id)
+
```

Comment:
Doesn't ExactMultiSet already inherit this from SketchBase?

Reply:
you are right, sorry, fixed it.